docs/Project.toml

```diff
@@ -12,3 +12,4 @@ Tulip = "6dd1b50a-3aae-11e9-10b5-ef983d2400fa"
 
 [compat]
 Documenter = "0.26"
+Literate = "0.28"
```

Comment:
before the version was `2.8`, now `0.28`. is this correct?

Reply:
no it isn't and I've got no idea how it could get here. :fearful: